src/components/GrowlNotification/index.js

```diff
@@ -29,24 +29,12 @@ const types = {
 const INACTIVE_POSITION_Y = -255;
 
 const PressableWithoutFeedback = Pressables.PressableWithoutFeedback;
+const translateY = new Animated.Value(INACTIVE_POSITION_Y);
```

Comment:
This value should be preserved in the internal state of the component

Reply:
why? it isn't changed

Reply:
This value changes when you apply Animated.spring(...) and is passed to GrowNotificationContainer as a prop

Reply:
The translateY is defined outside the functional component, which means that it will be shared across all instances of this component and will only be defined once, during the initialization of the module. If this value needs to be preserved for each individual instance of the component, it should be managed within the component's internal state or with a useRef hook. This approach ensures that the lifecycle of the translateY value is properly tied to each instance of the component, allowing for more predictable behavior and avoiding potential bugs.

Reply:
@ArekChr that's an excellent explanation 😄

Reply:
ok then - it makes sense! so I'll use `useRef` here as they do in docs